contracts/utils/StringUtils.sol

```diff
@@ -0,0 +1,65 @@
+pragma solidity ^0.6.2;
+
+library StringUtils {
+  // via https://github.com/oraclize/ethereum-api/blob/master/oraclizeAPI_0.5.sol
+  function strConcat(string memory _a, string memory _b, string memory _c, string memory _d, string memory _e) internal pure returns (string memory) {
```

Comment:
Basically, I think this method is not used in common, and suggest to include this library in specific project only.